### ISSUE TITLE
Create a basic page template

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#64]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

A basic page template with no special formatting.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by flyinmryan** [comment]
I would also like to see this completed.  In order to create a Page Template I believe the documentation requires an initial Page type be created with specific preferences set so I think that should be included as well.  Personally, I would like the see the process for creating templates be done programmatically as well, either from scratch or taken from a prior version site that's being upgraded.  Thanks
